go/interpreter/lfvm/instructions_test.go

```diff
@@ -772,40 +772,48 @@ func TestCreateShanghaiDeploymentCost(t *testing.T) {
 }
 
 func TestTransientStorageOperations(t *testing.T) {
+	address := tosca.Address{}
+	_, _ = rand.Read(address[:])
```

Comment:
this pattern is repeated in multiple tests, should we turn it into a utils fun? or use single explicit values such as `tosca.Address{1}; tosca.Key{2}`? Does it gain anything from being a fully random value rather than just a non zero initialized slice ?

Reply:
Yes, you are right. I chose this pattern because it was the fastest way to initialize all elements I thought of. 
I find the result much easier to debug, since the values are not 19/31 zeroes. And much faster to compare when printed out.

Reply:
I don't see the problem with turning it into a one liner, I just don't like seeing the same repeated line multiple times per tests, in many test. I agree that this should be used with care and never where it affects the control flow of the test.